crypto-onramp/src/main/java/com/stripe/android/crypto/onramp/OnrampInteractor.kt

```diff
@@ -124,14 +127,12 @@ internal class OnrampInteractor @Inject constructor(
         return when (result) {
             is LinkController.RegisterConsumerResult.Success -> {
                 val secret = consumerSessionClientSecret()
-                secret?.let {
-                    val permissionsResult = cryptoApiRepository
-                        .grantPartnerMerchantPermissions(it)
-
-                    permissionsResult.fold(
-                        onSuccess = { result ->
+                if (secret != null) {
+                    cryptoApiRepository.createCryptoCustomer(secret).fold(
+                        onSuccess = { customerResponse ->
```

Comment:
👀 Renamed `grantPartnerMerchantPermissions()` to `createCryptoCustomer()` to better match the request. Minor: also changed from `let-run` to `if-else` for readability